scripts/install/functions.sh

```diff
@@ -26,13 +44,16 @@ function cmd_clean {
 
     delete_dir ${BASE_DIR}
     delete_dir storage
-
-    yarn cache clean
+    if [[ "${KEEP_YARN_CACHE}" = false ]]; then
```

Comment:
first, not sure whether we want to keep yarn cache, because I think install script wants to install everything from scratch; second, using `yarn cache clean` to remove the shared cache files is better than force-removed some folder `~/.yarn-cache`.